sample/db/samples/addresses.rb

```diff
@@ -1,26 +1,23 @@
 united_states = Spree::Country.find_by!(iso: "US")
 new_york = Spree::State.find_by!(name: "New York")
 
-# Billing address
-Spree::Address.create!(
-  firstname: FFaker::Name.first_name,
-  lastname: FFaker::Name.last_name,
-  address1: FFaker::Address.street_address,
-  address2: FFaker::Address.secondary_address,
-  city: FFaker::Address.city,
-  state: new_york,
-  zipcode: 16_804,
-  country: united_states,
-  phone: FFaker::PhoneNumber.phone_number)
+first_names = ["Sterling", "Jennette", "Salome", "Lyla", "Lola", "Cheree", "Hettie", "Barbie", "Amelia", "Marceline", "Keeley", "Mi", "Karon", "Jessika", "Emmy"]
```

Comment:
Personally I prefer the `%w(Sterling Jennette)` syntax for arrays of strings. Also having a new line for each word is far better readable, but this should not block this PR.